nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenHTTP.java

```diff
@@ -95,6 +95,13 @@
         .expressionLanguageSupported(true)
         .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
         .build();
+    public static final PropertyDescriptor RETURN_CODE = new PropertyDescriptor.Builder()
+            .name("Return Code")
+            .description("The HTTP return code returned after every HTTP call")
+            .required(true)
+            .defaultValue("200")
```

Comment:
Probably should use string value of HttpServletResponse.SC_OK instead of "200".  Stays consistent with previous entry used.  Not a big deal at all obviously.